docs/templates/epacamd_eia_child.rst.jinja

```diff
@@ -0,0 +1,104 @@
+hard-coded into analyses.
+{% endblock %}
+
+{% block download_title %}
+{% endblock %}
+{% block download_docs %}
+{% endblock %}
+
+{% block availability_title %}
+What data is available through PUDL?
+------------------------------------
+{% endblock %}
+{% block availability %}
+PUDL imports all matched rows of the crosswalk, but retains only the id and year columns.
+{% endblock %}
+
+{% block respondents_title %}
+What facilities are included?
+-------------------------------------
+{% endblock %}
+{% block respondents %}
+The crosswalk includes boilers and generators with ids from the following agencies and data sets:
+
+* CAMD collects the Power Sector Emissions Data from fossil fuel-fired electric generating units (EGUs) over 25 MW in nameplate capacity.
+* EIA collects data on all EGUs (including nuclear and renewables) that are located at a plant that is over 1 MW in nameplate capacity and connected to the electricity grid.
+* FRS assigns facility identifiers to facilities that submit data to EPA under any program, including EGUs.
+* NEEDS combines the list of EGUs from both the Power Sector Emissions Data and EIA datasets, while assigning its own identifiers to those EGUs.
+{% endblock %}
+
+{% block original_data_title %}
+What does the original data look like?
+--------------------------------------
+{% endblock %}
+{% block original_data %}
+The `original EPA CAMD to EIA crosswalk <https://github.com/USEPA/camd-eia-crosswalk>`__
+was published by the US Environmental Protection Agency on GitHub. Catalyst forked their repository in order to maintain the code and issue yearly updates to the crosswalk on an ongoing basis.
+
+The crosswalk produced by that code links ids from the following agencies and data sets:
+
+* Environmental Protection Agency (EPA), and more specifically the Clean Air Markets Division (CAMD)
+* Energy Information Administration (EIA)
+* Facility Registry Service (FRS)
+* National Electric Energy Data System (NEEDS)
+
+The original data is a table which lists all boilers and generators in CAMD's database with their corresponding EIA boiler and generator if they have a match.
+IDs from FRS and NEEDS are included if respective matches were found.
+Identifying information for each boiler or generator is included as supporting evidence for the match and may help debug bad matches.
+
+The source code, along with a detailed methodology for how the crosswalk is computed, is available below:
+
+* `EPA's original script <https://github.com/USEPA/camd-eia-crosswalk>`__; most recent release October 2022
+* `Catalyst's fork <https://github.com/catalyst-cooperative/camd-eia-crosswalk-latest>`__
+* `Methodology <https://github.com/catalyst-cooperative/camd-eia-crosswalk-latest#methodology>`__
+* `Data dictionary <https://github.com/catalyst-cooperative/camd-eia-crosswalk-latest#output>`__
+{% endblock %}
+
+{% block notable_irregularities %}
+CAMD and EIA disagree on what a "unit" is
+-----------------------------------------
```

Comment:
this is a total nit but i don't think they disagree. they are just reporting on different slices of a plant (a generation unit for EIA and a smokestack unit for CAMD).. so maybe something like
```suggestion
CAMD and EIA report different types of "units"
----------------------------------------------
```